extras/keras-to-estimator/keras-to-estimator.ipynb

```diff
@@ -647,21 +652,21 @@
    "views": {}
   },
   "kernelspec": {
```

Comment:
I've found it works better to remove the kernel metadata.  Then when it loads it will just look for the user's available kernels. Sometimes if it's pre-specified that seems to cause problems for people.